serving/serving.go

```diff
@@ -381,3 +382,57 @@ func (serv *FeatureServer) getVectorTable(ctx context.Context, fv *metadata.Feat
 	}
 	return vectorTable, nil
 }
+
+func (serv *FeatureServer) ResourceLocation(ctx context.Context, req *pb.TrainingDataRequest) (*pb.ResourceFileLocation, error) {
+	// TODO: Modify this method to return the location of any resource within Featureform
+	// - This will require a change in the input to be a ResourceID (name, variant, and type)
+	// - Modifications to the Offline Store interface to pull the latest file location of a resource
+
+	id := req.GetId()
+	name, variant := id.GetName(), id.GetVersion()
+	serv.Logger.Infow("Getting the Resource Location:", "Name", name, "Variant", variant)
+
+	tv, err := serv.Metadata.GetTrainingSetVariant(ctx, metadata.NameVariant{Name: name, Variant: variant})
+	if err != nil {
+		return nil, errors.Wrap(err, "could not get training set variant")
+	}
+
+	if tv.Status() != metadata.READY {
```

Comment:
Do we want this logic? We're not pulling the training set here, just getting its location.

Reply:
we do need it because we need to pull the latest file for it.

Reply:
But this doesn't pull a file, it returns a filepath. Its fine but feel like the logic should be somewhere else